### PR TITLE
refactor: move Popup sign transaction component to its own file SignT…

### PR DESCRIPTION
**Pull request by PaulBernier**
…ransactionFormContent

Preparation work for https://github.com/project-serum/spl-token-wallet/pull/143

### CODE REVIEW

**Comment by armaniferrante** [comment]
Blocked by https://github.com/project-serum/spl-token-wallet/issues/147. Once that issue is addressed, we can merge this.

**Comment by PaulBernier** [comment]
@armaniferrante I have rebased and resolved conflicts again. I hope you can merge it before any other occurrence, thank you.

**Review by armaniferrante — APPROVED** [review]

**Pull request merged**
